### PR TITLE
US Election Changes

### PR DESCRIPTION
**Pull request by ccfox**
Completely overhauled the US election events, cleaning them up, adding more options to some, and adding dynamic results based on party popularity and certain flags.

I'll try to edit the loc to match these changes, since most of the loc also needs to be rewritten anyway as it either references certain events that may not have happened or otherwise has things that should be changed.

### CODE REVIEW

**Comment by ccfox** [comment]
I've changed most of the election locs, I'll locate the file the others are in and change them too - though in case it isn't done before the first I'd like to put forward what I have now.

Also included are two other minor loc changes, and a new event for Albania - since they were nominally a socialist republic up until early 1992. Its just a basic event for the elections that year tied to gov changes.

File: CWE/events/usa.txt [hunk 1]
```diff
@@ -476,27 +511,42 @@ country_event = {
...
+		}
 	}
 
 	option = {
 		name = EVT_8015840_B
 		#any_pop = { militancy = 2 consciousness = 6 } 
 		
 		any_pop = { 
-			limit = { strata = middle }
+			limit = { AND = { strata = middle }
```
Comment (settintotrieste) [inline CWE/events/usa.txt:528]: Extra bracket

Comment (ccfox) [reply CWE/events/usa.txt]: Fixed.

File: CWE/events/albania.txt [hunk 2]
```diff
@@ -242,3 +242,112 @@ country_event = {
...
+				factor = 10
+				ideology = communist_social
+			}
+			scaled_militancy = { 
+				factor = 5 
+				ideology = socialist
+			}
+		}
+		
+		ai_chance = { factor = 0.7 }
```
Comment (settintotrieste) [inline CWE/events/albania.txt:308]: Historical option should have a 90% ai chance.

File: CWE/events/usa_elections.txt [hunk 3]
```diff
@@ -1339,76 +2175,72 @@ country_event = {
...
 	}
 
 	mean_time_to_happen = { months = 11 modifier = { factor = 0.1 year = 2017 } }
 
 	option = {
-		name = "Joe Biden wins, as expected"
+		name = "Joe Biden (Moderate Democrat)"
+		
+		random_owned = { #This should give the player or AI the modifier if they don't have a majority support
+			limit = { owner = { AND = { government = democracy tag = USA NOT = { OR = {pop_majority_ideology = progressive pop_majority_ideology = big_tent pop_majority_ideology = liberal } } } } }
```
Comment (settintotrieste) [inline CWE/events/usa_elections.txt:2186]: OR = {pop_majority_ideology = progressive pop_majority_ideology = big_tent pop_majority_ideology = liberal }

Missing a space after OR = {

There are multiple other entries with this problem.

Comment (ccfox) [reply CWE/events/usa_elections.txt]: Fixed.

File: CWE/events/albania.txt (hunk 2 above)
Comment (ccfox) [reply CWE/events/albania.txt]: Alright.

**Pull request merged**